docs/salesforce.md

```diff
@@ -0,0 +1,162 @@
+# Using Snowfakery with Salesforce
+
+Snowfakery recipes that generate Salesforce records are just like any
+other Snowfakery recipes. You use SObject names for the 'objects'.
+There are several examples [in the Snowfakery repository](https://github.com/SFDO-Tooling/Snowfakery/tree/main/examples/salesforce)
+
+## Using Snowfakery within CumulusCI
+
+The process of actually generating the data into a Salesforce
+org happens through CumulusCI.
+
+[CumulusCI](http://www.github.com/SFDO-Tooling/CumulusCI) is a
+tool and framework for building portable automation for
+Salesforce projects. It is created by the same team that
+creates Snowfakery.
+
+The easiest way to learn about CumulusCI (and to learn how to
+install it) is with its [Trailhead Trail](https://trailhead.salesforce.com/en/content/learn/trails/build-applications-with-cumulusci).
+
+CumulusCI's documentation [describes](https://cumulusci.readthedocs.io/en/latest/data.html?highlight=snowfakery#generate-fake-data)
+how to use it with Snowfakery. Here is a short example:
+
+```s
+$ cci task run generate_and_load_from_yaml -o generator_yaml examples/salesforce/Contact.recipe.yml -o num_records 300 -o num_records_tablename Contact --org qa
+...
+```
+
+You can (and more often will) use generate_and_load_from_yaml from
+within a flow captured in in a `cumulusci.yml`.
+
+If you have CumulusCI configured and you would like to test this,
+you can do so by running some tests that are in the Snowfakery
+repo itself.
+
+The Snowfakery repo itself has a `cumulusci.yml` so it is a
+CumulusCI project. You can learn how to set up your own
+CumulusCI project by [studying it](https://github.com/SFDO-Tooling/Snowfakery/tree/main/cumulusci.yml)
+
+```s
+
+# clone the repo
+$ git clone https://github.com/SFDO-Tooling/Snowfakery.git
+# change working directory
+$ cd Snowfakery
+# run a specific test task
+$ cci task run generate_opportunities_and_contacts
+# run all of the test tasks and flows
+$ cci flow run test_everything
+...
+```
+
+## Record Types
+
+To specify a Record Type for a record, just put the Record Type’s API Name in a field named RecordType.
+
+```yaml
+# tests/cci/record_types.yml
+- object: Account
+  fields:
+    name: Bluth Family
+    RecordType: HH_Account
+- object: Account
+  fields:
+    name: Bluth Corporation
+    RecordType: Organization
+- object: Account
+  fields:
+    name: The Windors
+    RecordType: HH_Account
+- object: Account
+  fields:
+    name: The Firm
+    RecordType: Organization
+```
+
+## Creating and Referencing Person Accounts
+
+There are several features planned for the Salesforce Plugin, but
```

Comment:
This sentence makes it slightly unclear whether Person Account support is currently available or planned.

Reply:
https://github.com/SFDO-Tooling/Snowfakery/pull/353/commits/63e125ab927e8427ae9f5956554f86e8b0e842f8#diff-444f7dedc7f85729133c0374d624e8dacaf5d5d4b3d068064765275f73306683R79